### PR TITLE
Fix start timestamp (HPC-7503)

### PR DESCRIPTION
**Pull request by itkovian**

### CODE REVIEW

File: bin/sync_vsc_users.py [hunk 1]
```diff
@@ -90,7 +91,7 @@ def main():
     (last_timestamp, start_time) = retrieve_timestamp_with_default(
         SYNC_TIMESTAMP_FILENAME,
         start_timestamp=opts.options.start_timestamp,
-        default_timestamp="201710230000Z")
+        default_timestamp="201710230000")
```
Comment (wpoely86) [inline bin/sync_vsc_users.py]: why do you pass this argument at all? If has a default value, why not use that?

Comment (itkovian) [reply bin/sync_vsc_users.py]: Because we might not want to go back in time that much?

Comment (wpoely86) [reply bin/sync_vsc_users.py]: why not? If there is no file or option give, it should go back all the way no? 

On what basis did you pick this timestamp?

Comment (itkovian) [reply bin/sync_vsc_users.py]: Hmm, good question. I recall this being a point in time where we have all "correct" information, but we might want to regularly move this up? Actually, I've no good idea.

**Comment by wpoely86** [comment]
2019?

**Comment by itkovian** [comment]
nvm, dates are fixed in other PR :) was looking at the wrong branch. Thx @stdweird

File: bin/sync_vsc_users.py (hunk 1 above)
Comment (wpoely86) [reply bin/sync_vsc_users.py]: It depends on what you mean with correct :wink: 

If nothing is given, it should start at the beginning of time no?

Comment (stdweird) [reply bin/sync_vsc_users.py]: use an invalid timestamp (using none will probably pickup some other one.
default timestamp makes no sense. eitehr there is a timestamp file and it is used. or there is nt, and something is either very wrong or a new installation.

Comment (wpoely86) [reply bin/sync_vsc_users.py]: There is a default timestamp in vsc-utils anyway. Let's remove it here and you keep either file or a commandline option.

**Comment by itkovian** [comment]
See #84

**Pull request closed without merging**